tests/langtest_c.rs

```diff
@@ -52,7 +52,7 @@ fn run_suite(opt: &'static str) {
     };
 
     LangTester::new()
-        .comment_prefix("#")
+        .comment_prefix("##")
```

Comment:
Won't this break other tests than `truncate.newcg.c`? I really think this should be a PR in its own right.

Reply:
Your test was the only one using a comment :)

Reply:
OK but let's at least break it out into a separate commit later? Someone (OK, maybe me!) might want to see this distinctly in the history.

Reply:
Unfortunately I can't add the new debug info test without this change, so it's all dependent. It makes it possible to match the debuginfo lines that start with `#`.

Reply:
Yes, so when this comes to squashing, please make this a separate (first) commit.